utils/_context/_scenarios/stream_processing_offload.py

```diff
@@ -0,0 +1,131 @@
+import pytest
```

Comment:
I recommend adding ` r"utils/_context/_scenarios/stream_processing_offload\.py": scenario_groups.<good_group_name>,` in `compute_impacted_scenarios.py`, it'll speed up a lot the CI when you modify only this file